run-reports.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+export TASKCLUSTER_ROOT_URL=https://firefox-ci-tc.services.mozilla.com/
+export REPORT_SCHEDULER_ID="smoketest"
+export REPORT_PREFIX=https://github.com/taskcluster/mozilla-history/blob/master/WorkerVersions/
+
+# needs to be set in the environment:
+# TASKCLUSTER_CLIENT_ID
+# TASKCLUSTER_ACCESS_TOKEN
+
+set -euo pipefail
+
```

Comment:
Ah, didn't think about this usecase. I thought it will be part of the repo, so you could trigger it from the same location. 
I can add it

Reply:
That's true, it will be part of the same repo, but regardless, you would need to `cd` into the directory before running it for it to work, unless the `cd` is in the script itself...